libloki/proof-of-work.js

```diff
@@ -74,9 +74,9 @@ const pow = {
       ).toArrayBuffer()
     );
 
-    const nonceTrials =
-      _nonceTrials || (development ? DEV_NONCE_TRIALS : PROD_NONCE_TRIALS);
-    const target = pow.calcTarget(ttl, payload.length, nonceTrials);
+    const difficulty =
+      _difficulty || (development ? DEV_DIFFICULTY : PROD_DIFFICULTY);
```

Comment:
```
development ? DEV_DIFFICULTY : PROD_DIFFICULTY
```
Is there a need for this anymore?

Reply:
I left it there to cover the (hopefully impossible) case where `_difficulty` is somehow null or undefined
But now that you mention it I could probably just collapse the constants into a single fallback difficulty

The idea is that if you somehow broke your difficulty you would just make a request with the fallback difficulty and then the snode would inform you of the correct difficulty

Reply:
That would be a better idea.